addons/mail/static/src/components/message/message.xml

```diff
@@ -113,6 +113,14 @@
                                 </t>
                             </t>
                             <div class="o_Message_commands o_Message_headerCommands" t-att-class="{ 'o-mobile': env.messaging.device.isMobile }">
+                                <t t-if="props.hasUnfollowIcon and message.hasUnfollowIcon">
+                                    <span class="o_Message_command o_Message_commandUnfollow o_Message_headerCommand fa fa-bell-slash-o"
+                                        t-att-class="{
+                                            'o-message-selected': isSelected,
+                                            'o-mobile': env.messaging.device.isMobile,
+                                        }" t-on-click.stop="_onClickUnfollow" title="Unfollow"
```

Comment:
Why should it be stopped? Don't ever use `stopPropagation`. If you need to inform parent of event handled please use `isEventHandled`/`markEventHandled` combo explicitly.